design/arc/oneidentity-integration/sequence_diagram_system_characteristics.mmd

```diff
@@ -0,0 +1,16 @@
+sequenceDiagram
+    participant User-Agent
+    participant ARC
+    participant OneIdentity as One Identity
+
+    User-Agent->>ARC: Richiesta accesso all'applicazione
```

Comment:
I don't understand this action, is it a login action or access to protected resources?

Reply:
fixed the missunderstanding